Lib/test/test_memoryview.py

```diff
@@ -14,8 +14,10 @@
 import copy
 import pickle
 import struct
+import multiprocessing.shared_memory
```

Comment:
Can you defer this import until it's needed by the test (namely use a local import for threads and multiprocessing modules).

Reply:
Hi @picnixz ,

Thanks for your review! 😊

I have moved it inside the test case's local scope.

Best Regards,
Edward